app/views/posts/index.html.erb

```diff
@@ -1 +1,58 @@
-<div>Here is a list of posts for a given user</div>
\ No newline at end of file
+<div class="posts-container">
+  <div class="post-list-item">
+    <div class="user-photo">
+      <img src="<%= @user.photo %>" alt="user"/>
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <h4>Number of posts:
+        <%= @posts.count %></h4>
+    </div>
+  </div>
+
+  <div class="posts">
+    <% @posts.each do |post| %>
+      <%= link_to user_post_path(@user, post.id), class: 'post-item' do %>
+        <div class="post">
+          <div>
+            <h3><%= post.title %></h3>
+            <p><%= post.text.split(" ")[0..80].join(" ") %></p>
+          </div>
+          <div class="likes-comments">
+            <span class="comments">Comments:
+              <%= post.comments_counter %>,
+            </span>
+            <span class="likes">Likes:
+              <%= post.likes_counter %></span>
+          </div>
+        </div>
```

Comment:
- [ **OPTIONAL** ] For Rails best practices and DRY code rules, I think it would be better if you use `Partials file` for a `Post` card so you can render it every where you want. To help you you can refer **[this article for more details.](https://gorails.com/blog/how-to-use-partials-in-rails)**

Reply:
this too